packages/errors/src/errors.ts

```diff
@@ -382,6 +382,13 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/auto-cancellation-mismatch`
   },
+  CLOUD_BURN_IN_MISMATCH: (arg1: {runUrl: string, burnInConfig: string, configFileBaseName: string}) => {
+    return errTemplate`\
```

Comment:
This output is pretty compressed - more whitespace to have it formatted like the auto cancel mismatch error would make it more readable, I think